src/lightning/pytorch/strategies/xla.py

```diff
@@ -299,3 +280,13 @@ def register_strategies(cls, strategy_registry: Dict) -> None:
             cls,
             description=f"{cls.__class__.__name__}",
         )
+
+    def _pod_progress_bar_force_stdout(self) -> None:
+        # Why is it required? The way `pytorch_xla.distributed` streams logs
```

Comment:
pre-existing issue: this should check whether a TQDMPbar is present and active

Reply:
I can't really do that right now, I'd need access to a trainer reference.